lib/sales_analyst.rb

```diff
@@ -60,23 +60,23 @@ def average_item_price_for_merchant(merchant_id)
 
   def average_average_price_per_merchant
     sum_of_averages = all_merchants.sum do |merchant|
-      average_item_price_for_merchant(merchant.id) unless num_of_items_per_merchant[merchant] == 0
+      average_item_price_for_merchant(merchant.id) unless num_of_items_per_merchant[merchant].zero?
```

Comment:
oh cool, didn't know .zero? was a thing

Reply:
Tbh this must have been a change from @redferret maybe? I don't recall doing this and/or knowing that existed either 🤔